README.md

```diff
@@ -30,7 +30,7 @@ OpenCensus libraries require Go 1.8 or later as it uses the convenience function
 
 Tags represent propagated key values. They can propagated using context.Context
 in the same process or can be encoded to be transmitted on wire and decoded back
-to a TagSet at the destination.
+to a Set at the destination.
```

Comment:
I don't like this name for multiple reasons:
* It may be a Go thingy to shorter names but just Set does not make any sense (too generic).
* Consistency with other languages for the main classes names (this is probably one of the most important in Tags).
* Is not that bad that we duplicate 3 characters.

Reply:
The reason for this change is not the shortness. The call sites will always include the package name to refer to this type as `tags.Set`. It is clear that it is a tag set from the full name. We prefer not to repeat when naming, tags.TagSet doesn't make the tag set more recognizable.

Reply:
Maybe call it "tags.Set" in these top-level docs and it should read fine.